examples/advanced/variables.tf

```diff
@@ -13,6 +13,11 @@ variable "region" {
   description = "The IBM Cloud region where all resources will be deployed. Example values: 'us-south', 'eu-gb', 'au-syd'."
 }
 
+variable "zone" {
+  type        = string
```

Comment:
I think you should leave zone as a number, and add the var.region to make the full zone. That way you don't have a situation where var.region = "us-south" and you accidentally set this to "us-east-1" which won't work.